project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=0.13.16-M1
```

Comment:
> What is meant here by stability?

Most importantly, is whether 0.13.16 final is going to be source and--above all--binary backward compatible with 0.13.16-M1? Or is only backward compatible with 0.13.15, and could potentially break backward compat with 0.13.16-M1?

If we're going to publish an sbt plugin of Scala.js built on top of 0.13.16-M1, we need to make sure it will stay usable by builds using 0.13.16 final and later.

> So we could start using the cross-building functionality, without a full release.

I guess this joins the question above, but: what's the fundamental difference between the 0.13.16-M1 release and a "full" release? Is it just that you don't bother to write release notes and publish documentation for M releases, or are there underlying technical differences?